webapp/channels/src/components/channel_settings_modal/channel_settings_configuration_tab.test.tsx

```diff
@@ -337,4 +337,101 @@ describe('ChannelSettingsConfigurationTab', () => {
         const errorPanel = errorMessage.closest('.SaveChangesPanel');
         expect(errorPanel).toHaveClass('error');
     });
+
+    it('should save valid colors in hex format', async () => {
```

Comment:
the front end validation works great using the color input component. Added this two extra test to enhance the scenario coverage of the use of this component in the channel settings section.